src/project.rs

```diff
@@ -263,7 +394,77 @@ impl FuzzProject {
         cmd.arg("-minimize_crash=1")
             .arg(format!("-runs={}", tmin.runs))
             .arg(&tmin.test_case);
-        exec_cmd(&mut cmd).with_context(|| format!("could not execute command: {:?}", cmd))?;
+
+        let old_artifacts = self.get_artifact_paths(&tmin.target)?;
+
+        let mut child = cmd
+            .spawn()
+            .with_context(|| format!("failed to spawn command: {:?}", cmd))?;
+        let status = child
+            .wait()
+            .with_context(|| format!("failed to wait on child process for command: {:?}", cmd))?;
+        if !status.success() {
+            eprintln!("\n{:─<80}\n", "");
+            return Err(anyhow!("Command `{:?}` exited with {}", cmd, status))
+                .with_context(|| {
+                    "Test case minimization failed.\n\
+                     \n\
+                     Usually this isn't a hard error, and just means that libfuzzer\n\
+                     doesn't know how to minimize the test case any further while\n\
+                     still reproducing the original crash.\n\
+                     \n\
+                     See the logs above for details."
+                })
+                .into();
+        }
+
+        // Find and display the most recently modified artifact, which is
+        // presumably the result of minification. Yeah, this is a little hacky,
+        // but it seems to work. I don't want to parse libfuzzer's stderr output
+        // and hope it never changes.
+        let mut new_artifacts: Vec<_> = self
+            .get_artifact_paths(&tmin.target)?
+            .into_iter()
+            .filter(|a| !old_artifacts.contains(a))
+            .collect();
+        new_artifacts.sort_by_key(|p| {
```

Comment:
No need to sort, you can fold over a max function instead, it would be faster.

(Not that it matters much here)

Reply:
d'oh doesn't `Iterator` even have a `max` method?

Reply:
(yes, it does)

Reply:
why use that when you can instead use _>>functional programming&lt;&lt;_